content/foundations/physical_properties/physical_properties_conductivity.rst

```diff
@@ -31,7 +31,7 @@ Current density represents the amount of current flowing per unit cross-sectiona
 
 In essence, the electrical conductivity defines how easy it is for current to flow though a particular material when subjected to an electric field.
 In conductive materials (larger :math:`\sigma`), free charges move quite easily and strong currents may be induced by a relatively weak electrical field.
-In contrast, resistive materials (smaller :math:`\sigma`) require strong electrical fields in order to produce any significant current.
+In contrast, resistive materials (smaller :math:`\sigma`) require strong electrical fields or voltages in order to produce any significant current.
```

Comment:
@dccowan : can you connect with @sgkang and @dougoldenburg  on these edits? 

Thanks!